uefi-test-runner/examples/shell_params.rs

```diff
@@ -0,0 +1,60 @@
+// ANCHOR: all
```

Comment:
These `ANCHOR` comments are just used by mdbook (see https://rust-osdev.github.io/uefi-rs/HEAD/how_to/protocols.html#walkthrough for example), so unless you are looking to add this example to the book, you can leave these out.

Reply:
I think it would be good to have a shell application example in the book.
However it would be more useful once the shell protocol is merged.